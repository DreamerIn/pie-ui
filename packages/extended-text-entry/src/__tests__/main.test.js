import React from 'react';
import Main from './../main'

describe('Render Main Component', () => {
  let wrapper;

  let model = {width: 100, height : 50};
  beforeEach(() => {
    wrapper = shallow(<Main model={model} onChange={() => {}} />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
});