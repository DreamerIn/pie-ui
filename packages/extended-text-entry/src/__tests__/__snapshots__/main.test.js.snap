// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Main Component should match snapshot 1`] = `
<Main
  classes={
    Object {
      "main": "Main-main-1",
      "prompt": "Main-prompt-2",
    }
  }
  model={
    Object {
      "height": 50,
      "width": 100,
    }
  }
  onChange={[Function]}
  session={
    Object {
      "value": "hi",
    }
  }
/>
`;
